dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/group/ComponentGroup.kt

```diff
@@ -71,6 +74,25 @@ class ComponentGroup(
         }
     }
 
+    override fun generateDefaultUploadConfig(sectionUploadConfigBuilder: SectionUploadConfigBuilder) {
+        val localLabel = label
+        require(!localLabel.isNullOrBlank()) {
+            "You must specify a label for the group $identifier to generate a view configuration"
+        }
+        val containerSection = sectionUploadConfigBuilder.addSection(
+            identifier = identifier,
+            label = localLabel,
+            labelBadgeColor = uploadPageLabelBadgeColor,
+            expandOnPageLoad = viewPageExpandOnPageLoad,
```

Comment:
`expandOnPageLoad` does not really make sense for the upload page (or does it?)

Reply:
No it doesn't.

Reply:
Removed